pkg/storage/export/export/export.go

```diff
@@ -53,7 +52,9 @@ import (
 	"kubevirt.io/kubevirt/pkg/certificates/triple"
 	"kubevirt.io/kubevirt/pkg/certificates/triple/cert"
 	"kubevirt.io/kubevirt/pkg/controller"
-	"kubevirt.io/kubevirt/pkg/instancetype"
+	instancetypeexpand "kubevirt.io/kubevirt/pkg/instancetype/expand"
+	instancetypefind "kubevirt.io/kubevirt/pkg/instancetype/find"
```

Comment:
nit: Use camel case like for `preferenceFind` below?

Reply:
ops, as discussed camel case for import names is actually a linting error if this code was being linted. I'll remove it from `preferenceFind` if I respin the series.